Android/res/values-es/strings.xml

```diff
@@ -0,0 +1,454 @@
+    <string name="drawer_open">Abrir el apartado de navegación</string>
+    <string name="drone_autopan_long_press">Haz una pulsación larga para activar el autoencuadre del drone</string>
+    <string name="drone_no_location">La localizacion del drone no está disponible</string>
+    <string name="dsinfo">Droneshare es un servicio web gratuito que le permite revisar, analizar y compartir los vuelos.</string>
+    <string name="editor">Editor</string>
+    <string name="editor_err_land_rtl_added">No puedes añadir waypoints despues LAND or RTL</string>
+    <string name="editor_info_window">Distancia: 0m, Tiempo de vuelo: 0:00</string>
+    <string name="editor_info_window_distance">Distancia: %s</string>
+    <string name="editor_info_window_flight_time">Tiempo de Vuelo: %1$02d:%2$02d</string>
+    <string name="editor_info_window_heading">Cabeceo: %.0f°</string>
+    <string name="editor_multi_edit_no_waypoint_error">No has seleccionado Waypoints</string>
+    <string name="email_address">Dirección de mail</string>
+    <string name="empty_content" translatable="false">--</string>
+    <string name="error_when_opening_file">Error al abrir el fichero</string>
+    <string name="exception_draw_1_more_polygon_points">Pon 1 punto mas</string>
+    <string name="exception_draw_2_more_polygon_points">Pon 2 puntos mas</string>
+    <string name="exception_draw_polygon">Pinta un poligono de Waypoints</string>
+    <string name="expandable_listview_icon_desc">Icono ListView Expandible</string>
+    <string name="export_to_device">Exporta un fichero</string>
+    <string name="file_saved_error">Error al guardar el fichero</string>
+    <string name="failed_start_calibration_message">No se puede empezar la calibracion</string>
+    <string name="file_saved_success">Fichero guardado</string>
+    <string name="firmware_version">Firmware-Version</string>
+    <string name="flight_altitude">Altura de vuelo</string>
+    <string name="flight_data">Datos de vuelo</string>
+    <string name="flight_modes_label">Modos de vuelo</string>
+    <string name="footprint">Marca</string>
+    <string name="ground_resolution">GSD</string>
+    <string name="guided_mode_warning">"Pulsa OK para entrar en el modo guiado(GUIDED). \nPara cambiar la localizacion, deja pulsado el mapa.</string>
+    <string name="ground_speed_label">Velocidad en tierra</string>
+    <string name="guided_scan_roi_set_message">Poniendo la localización seleccionada como objetivo</string>
+    <string name="help">Ayuda</string>
+    <string name="help_item_description"></string>
+    <string name="import_from_device">Importar fichero</string>
+    <string name="inner_wps">WPs cercanos</string>
+    <string name="label_advaned">Avanzado</string>
+    <string name="label_altitude_step"></string>
+    <string name="label_enter_filename">Poner nombre del fichero</string>
+    <string name="label_login">Loguear</string>
+    <string name="label_logout">Desloguear</string>
```

Comment:
"Cerrar Sesión", would this change break the layout?, also do you agree with this change? (English to keep others in the conversation)